sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -562,37 +642,51 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         "license" -> JSON.str("UNLICENSED")
       ),
 
-      npmUpdate := NpmUpdateTasks.npmUpdate(
-      baseDirectory.value,
-      (crossTarget in npmUpdate).value,
-      scalaJSBundlerPackageJson.value.file,
-      useYarn.value,
-      scalaJSNativeLibraries.value.data,
-      streams.value,
-      npmExtraArgs.value,
-      yarnExtraArgs.value),
+      npmUpdate := npmInstall.value,
+
+      crossTarget in npmInstall := (crossTarget in npmUpdate).value,
+
+      npmInstall := {
+        val dir = npmInstallDependencies.value
+        val _ = npmInstallJSResources.value
+        dir
+      },
+
+      npmInstallDependencies := NpmUpdateTasks.npmInstallDependencies(
+        baseDirectory.value,
+        (crossTarget in npmInstall).value,
+        scalaJSBundlerPackageJson.value.file,
+        useYarn.value,
+        streams.value,
+        npmExtraArgs.value,
+        yarnExtraArgs.value),
+
+      npmInstallJSResources := NpmUpdateTasks.npmInstallJSResources(
+        (crossTarget in npmInstall).value,
+        scalaJSNativeLibraries.value.data,
+        streams.value),
 
       scalaJSBundlerPackageJson :=
         PackageJsonTasks.writePackageJson(
-          (crossTarget in npmUpdate).value,
+          (crossTarget in npmInstall).value,
           npmDependencies.value,
           npmDevDependencies.value,
           npmResolutions.value,
           additionalNpmConfig.value,
-          fullClasspath.value,
+          dependencyClasspath.value,
           configuration.value,
           (version in webpack).value,
           (version in startWebpackDevServer).value,
           webpackCliVersion.value,
           streams.value
         ),
 
-      crossTarget in npmUpdate := {
+      crossTarget in npmInstall := {
```

Comment:
We should document that `crossTarget in npmUpdate` has to be replaced with `crossTarget in npmInstall` in the changelog. Also, since this is a major breakage we should probably bump the version number to 0.14.0.

Reply:
Having both this definition and `crossTarget in npmInstall := crossTarget in npmUpdate.value` looks wrong. I think the most migration friendly strategy is to keep `npmUpdate` and `crossTarget in npmUpdate` as they were, and then define `npmInstall := npmUpdate.value` and `crossTarget in npmInstall := (crossTarget in npmUpdate).value`. This way, users that have overridden `npmUpdate` or `crossTarget in npmUpdate` will still have something that works (with deprecation warnings).